apps/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ include_directories(${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_BINARY_DIR})
 add_subdirectory(monsteerGUI)
 
 set(MONSTEER_SPIKE_RECEIVER_SOURCES monsteer_spike_receiver.cpp)
-set(MONSTEER_SPIKE_RECEIVER_LINK_LIBRARIES Brion)
+set(MONSTEER_SPIKE_RECEIVER_LINK_LIBRARIES Brion Monsteer)
```

Comment:
It only includes one header to get MONSTEER_BRION_SPIKES_PLUGIN_SCHEME, but doesn't need to link against Monsteer. I guess it doesn't matter so much since it's an app, not a lib which might be reused.